### ISSUE TITLE
System Error when running Ganache 2.5.4 on win32

### ISSUE THREAD

**Issue by khomohzie** [trufflesuite/ganache-ui#4793]
PLATFORM: win32
GANACHE VERSION: 2.5.4

EXCEPTION:
```
TypeError: Cannot read property 'stack' of undefined
    at IpcMainImpl.<anonymous> (C:\Program Files\WindowsApps\GanacheUI_2.5.4.0_x64__5dg5pnz03psnj\app\resources\app.asar\webpack:\src\integrations\ethereum\index.js:96:31)
```

I used the truffle console to send a transaction via the web3 API i.e `web3.eth.sendTransaction(from: "", to: "", data: "", value: "")`

1. The issue here is for the data property, I'm using the function signature of my addFunds() method which runs just fine. But when I put in a wrong function signature e.g instead of 0xa26759cb, I input 0xa259cb, it throws an error on the console saying the evm reverted the transaction but ends up showing on the Ganache UI. Now when I click on the transaction in Ganache, I receive the system error above.
2. I decide to run the send transaction function without the data property. And I expect it to use the receive function I wrote in my contract i.e `receive() external payable {}`. This one runs and doesn't throw an error in the console. The transaction also shows on Ganache UI but again, when I click it, I receive the system error above.

In summary, only when I put in the correct function signature in the data property will I be able to open the transaction details on Ganache without an error.

**NOTE**: On Ubuntu, using WSL, these 2 scenarios ran successfully without any errors at all.
